web/src/components/DandiAppBar.vue

```diff
@@ -171,7 +171,7 @@ export default {
       return !(this.name && this.description);
     },
     ...mapGetters(['loggedIn', 'user']),
-    ...mapState(['apiKey', 'girderRest']),
+    ...mapState(['apiKey', 'girderRest', 'status']),
```

Comment:
Is having `status` here a mistake? It's not in the store and isn't used here either.

Reply:
Yeah, I forgot to take it out. It's supposed to be in #137 branch that I'm working on rn.

Reply:
Cool, once that's removed I think this looks good.

Reply:
Once it passes all the tests, can I merge and close the issue or should I wait?

Reply:
Yep, for the record #138.